### DISCUSSION TITLE
How can I customize the starship?

### DISCUSSION THREAD

**Discussion by jvzaniolo** [starship/starship#1907] [Q&A] [answered]
I used to have spaceship with this config in .zsh:

```
SPACESHIP_PROMPT_ORDER=(
  user          # Username section
  dir           # Current directory section
  host          # Hostname section
  git           # Git section (git_branch + git_status)
  hg            # Mercurial section (hg_branch  + hg_status)
  exec_time     # Execution time
  line_sep      # Line break
  vi_mode       # Vi-mode indicator
  jobs          # Background jobs indicator
  exit_code     # Exit code section
  char          # Prompt character
)
SPACESHIP_USER_SHOW=always
SPACESHIP_PROMPT_ADD_NEWLINE=false
SPACESHIP_CHAR_SYMBOL="❯"
SPACESHIP_CHAR_SUFFIX=" "
```

Is there a way to make it look like this?

![Captura de tela 2020-11-19 155736](https://user-images.githubusercontent.com/54036572/99710972-f84de600-2a7f-11eb-9e27-17f233613a2b.png)

**Comment by mickimnet** [comment, marked as answer]
Yes, you can create and edit a `starship.toml` file -- see [docs:configuration](https://starship.rs/config/) for a how to and  download e. g. the `starship.toml` from the _Nerd Font Symbols_ configuration as a base for changes--currently available in the [preview of the upcoming gallery](https://deploy-preview-1829--starship-rs.netlify.app/gallery/#contribution-to-the-gallery).
